### PR TITLE
Fixed FMOD volume sync going max volume if source distance is out of range

### PR DESCRIPTION
**Pull request by Measurity**

### CODE REVIEW

File: NitroxClient/GameLogic/FMOD/FMODSystem.cs [hunk 1]
```diff
@@ -11,19 +12,16 @@ namespace NitroxClient.GameLogic.FMOD
...
         {
             this.packetSender = packetSender;
-
-            string soundsWhitelist = Properties.Resources.soundsWhitelist;
-
+            string soundsWhitelist = Resources.soundsWhitelist;
             if (string.IsNullOrWhiteSpace(soundsWhitelist))
             {
-                Log.Error(new NullReferenceException(), "[FMODSystem]: soundsWhitelist.csv is null or whitespace");
+                Log.Error(new Exception(), "[FMODSystem]: soundsWhitelist.csv is null or whitespace");
```
Comment (Jannify) [inline NitroxClient/GameLogic/FMOD/FMODSystem.cs:24]: Why changing it to a generic exception? If NRE isn't fitting I would remove it completely and only log the message.

Comment (Measurity) [reply NitroxClient/GameLogic/FMOD/FMODSystem.cs:24]: Because it's null or whitespace, not just null. But still wanted the stack trace.

Comment (Measurity) [reply NitroxClient/GameLogic/FMOD/FMODSystem.cs:24]: The stack trace is useless, with this `new` on same line. Will remove it.

File: NitroxClient/MonoBehaviours/MultiplayerSeaMoth.cs [hunk 2]
```diff
@@ -24,9 +24,10 @@ protected override void Awake()
 
         protected void Update()
         {
+            // Clamp volume between 0 and 1 (nothing or max). Going below 0 turns up volume to max.
             float distance = Vector3.Distance(Player.main.transform.position, transform.position);
-            rpmSound.GetEventInstance().setVolume(1 - distance / radiusRpmSound);
-            revSound.GetEventInstance().setVolume(1 - distance / radiusRevSound);
+            rpmSound.GetEventInstance().setVolume(Mathf.Clamp01(1 - distance / radiusRpmSound));
```
Comment (Jannify) [inline NitroxClient/MonoBehaviours/MultiplayerSeaMoth.cs:29]: Would do the same in `NitroxServer/Communication/Packets/Processors/PlayFMODAssetProcessor.cs`.

**Review by Jannify — CHANGES_REQUESTED** [review]

**Review by Jannify — APPROVED** [review]

**Pull request merged**